build/Tests/FunctionalTests/FunctionalTests.vcxproj

```diff
@@ -295,8 +293,12 @@
   <ItemGroup>
     <Media Include="$(MSBuildThisFileDirectory)..\..\..\tests\functionaltests\Tests\AssetsLibraryTests\AssetsLibraryTestVideo.mp4" />
   </ItemGroup>
+  <ItemGroup>
+    <UXResources Include="$(MSBuildThisFileDirectory)..\..\..\samples\XAMLCatalog\XAMLCatalog\Images\*" />
```

Comment:
I'll take a shot at it - VS wasn't happy when I specified them as resources/content so I followed the same pattern used by CTDrawingTests.